frontend/app/model/FeatureChoice.scala

```diff
@@ -29,11 +28,20 @@ object FeatureChoice {
 
 case object Books extends FeatureChoice {
   override val zuoraCode = "Books"
-  override val label = "4 free books"
+  override val label = "4 books"
+  override val description = """
+    |Get six Guardian Live tickets to use throughout the year.
+    |Use one ticket per event at the event of your choosing.
+  """.stripMargin
 }
+
 case object FreeEventTickets extends FeatureChoice {
   val allowance = zuoraFreeEventTicketsAllowance
   override val zuoraCode = "Events"
-  override val label = s"$allowance free events"
+  override val label = s"$allowance tickets"
+  override val description = """
```

Comment:
Also, is there a reason why we use a word in one description and a number in the other? If we decide to go for numbers, can we please render the `$allowance` value into the description? I have made this value configurable to facilitate QA work